src/Firebase/AppCheck/ApiClient.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kreait\Firebase\AppCheck;
+
+use Beste\Json;
+use GuzzleHttp\ClientInterface;
+use Kreait\Firebase\Exception\AppCheckApiExceptionConverter;
+use Kreait\Firebase\Exception\AppCheckException;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\UriInterface;
+use Throwable;
+
+/**
+ * @internal
+ *
+ * @phpstan-type AppCheckTokenShape array{
+ *     token: string,
+ *     ttl: string
+ * }
+ */
+class ApiClient
+{
+    private ClientInterface $client;
+    private AppCheckApiExceptionConverter $errorHandler;
+
+    public function __construct(ClientInterface $client)
+    {
+        $this->client = $client;
+        $this->errorHandler = new AppCheckApiExceptionConverter();
+    }
+
+    /**
+     * @throws AppCheckException
+     *
+     * @return AppCheckTokenShape
+     */
+    public function exchangeCustomToken(string $appId, string $customToken): array
+    {
+        $response = $this->requestApi('POST', 'apps/'.$appId, [
```

Comment:
This request results in a 404 - according to the [documentation](https://firebase.google.com/docs/reference/appcheck/rest/v1/projects.apps/exchangeCustomToken#http-request), the target URL should be `apps/<appId>:exchangeCustomToken`, how did you make this work in your tests?